core/templates/pages/terms-page/terms-page.mainpage.html

```diff
@@ -117,7 +117,7 @@ <h2>Hosted Created Content and Intellectual Property</h2>
               </p>
 
               <a class="oppia-privacy-terms-anchor" name="refraining-from-certain-activities"></a>
-              <h2 style="margin-top: 40px;">Refraining from Certain Activities</h2>
+              <h2 class="oppia-refrain-from-activity">Refraining from Certain Activities</h2>
```

Comment:
Make this a noun. E.g. oppia-refrain-from-activity-header. Ditto for your other class names.

That said, this seems like an extremely special case -- styling should be applied uniformly. If you get rid of the additional style, does the page look bad? If not then getting rid of it is fine, but please check to ensure that there are no unintended visual side effects.

Reply:
Done.